client/components/mma/upgrade/UpgradeSupportAmountForm.tsx

```diff
@@ -228,7 +228,7 @@ export const UpgradeSupportAmountForm = ({
 							threshold={threshold}
 						/>
 					)}
-					{!continuedToConfirmation && chosenAmount && !errorMessage && (
+					{!continuedToConfirmation && !errorMessage && chosenAmount && (
```

Comment:
```suggestion
					{!continuedToConfirmation && !errorMessage && !!chosenAmount && (
```

changing the order here seems to work but you could add this for clarity. To be clear, chosenAmount is still on the left side of this && with regard to the react element (but now the error message would be set when the amount is 0)